tests/unit/amo/reducers/test_guides.js

```diff
@@ -0,0 +1,30 @@
+import guidesReducer, {
+  fetchGuidesAddons,
+  FETCH_GUIDES_ADDONS,
+  initialState,
+} from 'amo/reducers/guides';
+import { LOAD_ADDON_RESULTS } from 'core/reducers/addons';
+
+describe(__filename, () => {
+  describe('reducer', () => {
+    it('initializes to its default state', () => {
+      const state = guidesReducer(undefined, {});
+      expect(state).toEqual(initialState);
+    });
+
+    it('updates the loading flag status', () => {
+      const state = guidesReducer(
+        undefined,
+        fetchGuidesAddons({ guid: 'test', errorHandlerId: 'test' }),
+      );
+
+      const newState = guidesReducer(state, { type: FETCH_GUIDES_ADDONS });
```

Comment:
I am not sure I understand why you did that? You should call `fetchGuidesAddons` here, but it is done 2 lines above.

Reply:
yep, you're right 🤦‍♀️